### PR TITLE
add ignore file for manager

### PR DESCRIPTION
**Pull request by jichenjc**
the cmd/manager/manager is exec file and should not in git tree

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: cmd/manager/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+manager
```
Comment (gyliu513) [inline cmd/manager/.gitignore]: I think we already ignored this binary in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/.gitignore#L139 , it should be located in `bin/manager`?

Comment (jichenjc) [reply cmd/manager/.gitignore]: I don't know, but I saw this manager in my cmd/manager folder now while definitely I didn't create the manger by running 'go build', so not sure where it comes from...

Comment (gyliu513) [reply cmd/manager/.gitignore]: I usually use `make manager` in top dir.

Comment (jichenjc) [reply cmd/manager/.gitignore]: ok, let's close this ,if I meet this again I will reopen it , thanks for the commetns

**Pull request closed without merging**